src/scenes/Languages/Detail/LanguageDetail.tsx

```diff
@@ -104,45 +116,72 @@ export const LanguageDetail = () => {
               </Fab>
             ) : null}
           </div>
-          <Grid container spacing={2}>
+          <Grid container spacing={2} alignItems="center">
             <Grid item>
-              <Sensitivity value={language?.sensitivity} loading={!language} />
+              <Sensitivity value={sensitivity} loading={!language} />
             </Grid>
             <BooleanProperty
               label="Dialect"
               redacted="You do not have permission to view whether the language is a dialect"
-              data={language?.isDialect}
+              data={isDialect}
               wrap={(node) => <Grid item>{node}</Grid>}
             />
             <LeastOfThese language={language} />
+            <BooleanProperty
+              label="Sign Language"
+              redacted="You do not have permission to view whether the language is a sign language"
+              data={isSignLanguage}
+              wrap={(node) => <Grid item>{node}</Grid>}
+            />
           </Grid>
           <DisplayProperty
             label="Pronunciation Guide"
-            value={language?.displayNamePronunciation.value}
+            value={displayNamePronunciation?.value}
             loading={!language}
           />
           <DisplayProperty
-            label="Ethnologue Code"
-            value={
-              ethnologue?.code.value ?? ethnologue?.provisionalCode.value
-                ? `${ethnologue.provisionalCode.value} (provisional)`
-                : null
-            }
+            label="Ethnologue Name"
+            value={ethnologue?.name.value}
             loading={!language}
           />
+          {isSignLanguage?.value && signLanguageCode?.value ? (
+            <DisplayProperty
+              label="Sign Language Code"
+              value={signLanguageCode.value}
+              loading={!language}
+            />
+          ) : (
+            <DisplayProperty
+              label="Ethnologue Code"
+              value={
+                ethnologue?.code.value ?? ethnologue?.provisionalCode.value
```

Comment:
Can we fix this while we're in here? If the `code` has a value but the `provisionalCode` doesn't, this will display "null":

![image](https://user-images.githubusercontent.com/728777/94489101-df2c6580-01b1-11eb-920c-3126dcdca72d.png)

Reply:
While we are here we also want to display both the eth code **and** provisional code. I was mistaken previously to try to pick one or the other.

Reply:
good catch, fixed

Reply:
@CarsonF didnt see your comment earlier.  To confirm Ethnologue Code is conditional based on 

> While we are here we also want to display both the eth code **and** provisional code. I was mistaken previously to try to pick one or the other.

@CarsonF didn't see this.  Just to confirm, Ethnologue Code has that conditional relationship with Sign Language Code as discussed, but Provisional Code is independent of the other two?